### ISSUE TITLE
Support for copy and paste

### ISSUE THREAD

**Issue by jan-ladleif** [jan-ladleif/chor-js#69]
Copy and paste does not work currently. For example, when trying to paste a choreography task:

```
TypeError: self.canPaste is not a function[Learn More] ChoreoRules.js:213:11
	[38]</ChoreoRules.prototype.init/< ChoreoRules.js:213:11
	unwrapEvent/< CommandInterceptor.js:43:11
	<anonymous> self-hosted:978:17
	invokeFunction EventBus.js:480:9
	[172]</EventBus.prototype._invokeListener EventBus.js:356:18
	[172]</EventBus.prototype._invokeListeners EventBus.js:344:18
	[172]</EventBus.prototype.fire EventBus.js:304:18
	[167]</CommandStack.prototype._fire CommandStack.js:346:13
	[167]</CommandStack.prototype.canExecute CommandStack.js:178:15
	[278]</Rules.prototype.allowed Rules.js:41:14
	[201]</CopyPaste.prototype.paste CopyPaste.js:225:13
	[210]</EditorActions.prototype._registerDefaultActions/< EditorActions.js:101:6
	[210]</EditorActions.prototype.trigger EditorActions.js:173:9
	[221]</KeyboardBindings.prototype.registerBindings/< KeyboardBindings.js:107:6
	invokeFunction EventBus.js:480:9
	[172]</EventBus.prototype._invokeListener EventBus.js:356:18
	[172]</EventBus.prototype._invokeListeners EventBus.js:344:18
	[172]</EventBus.prototype.fire EventBus.js:304:18
	[220]</Keyboard.prototype._keyHandler Keyboard.js:94:19
	<anonymous> self-hosted:976:17
```

**Comment by yT0n1** [comment]
First of all we will have to add a canPaste function to ChoreoRules. Then BpmnCopyPaste.js which registers a function on 'element.paste' (element without 's', so every single element) looks like a candidate to overwrite/disable/subscribe. 
It iteratively gets a context which contains some (should be possible to contain all) pasted elements. It could be possible to implement correct copy/clone and re-referencing here.

**Comment by yT0n1** [comment]
Closed in #81
